src/bin/cargo/commands/test.rs

```diff
@@ -41,9 +41,9 @@ pub fn cli() -> Command {
             "Test only the specified example",
             "Test all examples",
             "Test only the specified test target",
-            "Test all test targets",
+            "Test all targets that have the `test = true` manifest flag set",
             "Test only the specified bench target",
-            "Test all bench targets",
+            "Test all targets in benchmark mode that have the `bench = true` manifest flag set",
```

Comment:
> I am not sure what it means to "test a target in benchmark mode",

I don't really know, either. I think that should be removed in man pages. Basically how I understand it is collecting Cargo targets that have `bench = true` and nothing else.

Reply:
Would be good to confirm that before adjusting the docs.
Seems like this was added in https://github.com/rust-lang/cargo/commit/f78ab2ca4b64f8160c21738629e5b362d3ab8310, so Cc @ehuss

Reply:
I think "in benchmark mode" is left over from when cargo used to change modes/profiles based on which targets were being built. When we stabilized named profiles, we also changed cargo to stop doing that and use only a single profile.

I would delete it here and in the docs.

Reply:
Okay, I have done that. This was duplicated in a bunch of places so I hope this will work.

The term `benchmark mode` is now gone entire; `test mode` still appears in a few places -- probably better for one of you to take a look since I am lacking the necessary context here.